docs/explanations/about_esm.md

```diff
@@ -61,3 +61,20 @@ You can manage `esm-infra` and `esm-apps` using `pro` on the command line. To
 find out how, read our guide on
 [enabling and disabling these services](../howtoguides/enable_esm_infra.md)
 on your machine.
+
+## Are `esm-infra` and `esm-apps` packages preferred over regular updates?
+
+Yes. The Pro Client will deliver the following configuration files to apt to
+ensure the `esm` updates are preferred when running `apt upgrade`:
+
+- `/etc/apt/preferences.d/ubuntu-pro-esm-infra`
+- `/etc/apt/preferences.d/ubuntu-pro-esm-apps`
+
+Those files are *pinning* the respective `esm` origins to 510 - a value which
+is higher than the standard (500) defined in apt. Although the files will
```

Comment:
Do we need to include the values? Because without already knowing what pinning is or how it works, the values could be really any values and hold no meaning for the reader. I think that if you want to include the values, we will need to explain what pinning is (however briefly). I'm also not 100% sure what you mean by "esm origins". It seems to imply there's more than one place to get esm updates from. What packages are supplied at the standard 500? (i.e., what is pinning esm to 510 giving preference over?)

Reply:
I removed specificities (as origins or values) and tried to better explain what is the impact about the pinning.
Please let me know if there is any more improvement needed.